packages/botonic-core/src/handoff.js

```diff
@@ -172,6 +172,23 @@ export async function getAvailableAgents(session) {
   return resp.data
 }
 
+export async function getAgentVacationsRanges(
+  session,
+  { agentId, agentEmail }
+) {
+  const baseUrl = session._hubtype_api || 'https://api.hubtype.com'
```

Comment:
nit: Ideally the URL string should be a constant or an env var

Reply:
yes, good catch, put it in a constant! 👍